### ISSUE TITLE
Some question

### ISSUE THREAD

**Issue by sevLax001** [idelac3/RFBService#2]
i have 3 pc 
first pc connected 
`RFBServiceExample` 
`final RFBService service = new RFBService(5500);`
CONNECTED

2nd pc connected 
`RFBServiceExample`  
` final String listeningPort = "5500";`
`final String connectTo = "first pc public address";`
`final RFBService service = new RFBService();` 
CONNECTED

now my question how to i access first pc on vnc viewer and 2nd pc vnc viewer

thanks

**Comment by sevLax001** [comment]
let me add this how do i view 2nd pc screen from pc 1

**Comment by idelac3** [comment]
Use VNC viewer to connect to RFB service.

It is not possible to use RFB service as relay or similar.

**Comment by sevLax001** [comment]
yes i know am going to use VNC viewer  but i cant know how to connect to 2nd pc that is issue 
please can you explain more about this function

```final String listeningPort = "5500";
final String connectTo = "first pc public address";
final RFBService service = new RFBService();```

am trying to working on viewer that list all connected client and i can click any of client from my viewer for now on the first pc working the 2nd pc that connected to first pc using `connectTo` i thought the first will generate separate port for 2nd pc since it connected through first pc public ip and port

**Comment by idelac3** [comment]
RFB service is a server-side impl. of RFB protocol. That means it will serve desktop to VNC clients (viewers) just like any other VNC server/service.

In client-server world, usually client initiates TCP connection to server. Eg. web browser will always open TCP connection to web server. Never opposite.

VNC is a bit different: VNC viewers have option to work in 'listening' mode, thus allowing VNC servers to connect to viewer. Most of them have 'listening' mode. That might confuse, but VNC server will always provide desktop (it will never become VNC viewer), no matter who connects to whom.

So, if you start your VNC viewer in listening mode, accepting sessions from VNC server(s), then you can tell RFB service IP address (and optionally TCP port if its different than 'default' TCP port 5500 for listening mode), and RFB service should connect to your viewer and serve desktop image.

Note also that it is impossible to connect one RFB service to another RFB service. That simply makes no sense.

It does not matter if you have more than 1 PC running RFB service (or any VNC service implementation), your VNC viewer should be capable handling more than 1 desktop session.
